CONTRIBUTING.md

```diff
@@ -2,59 +2,64 @@
-    ```
-    [commit]
-        gpgsign = true
-    [user]
-        email = <Your GH account email>
-        name = <Your Name>
-        signingkey = <your_signing_key>
-    [gpg]
-        program = gpg
-    ```
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template.
-1. An Expensify engineer will be automatically assigned to review your PR
-1. You will need all checks to pass:
-	1. CLA - You must sign our [Contributor License Agreement](https://github.com/Expensify/Expensify.cash/blob/master/CLA.md) by following the CLA bot instructions that will be posted on your PR
-	1. Tests - All tests must pass before a merge of a pull request
-	1. Lint - All code must pass lint checks before a merge of a pull request
-
-#### Testing
-Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, and Android) to confirm the fix works as expected and there are no regressions.
-
-#### Review Tips
-
-### Getting Attention
-- If you have made a change and are ready for another review the best way to get your reviewer's attention is to leave a comment that says "Updated" on the PR itself.
-- Please do NOT ping individual reviewers in the Slack channel to get their attention and keep the conversation in GitHub whenever possible.
-- Patience is a virtue. Reviews can sometimes take place over the course of several days. If your PR has not been addressed after 3 days please leave a comment on the PR. If the PR goes without a response for another day please let us know via Slack.
-
-### JavaScript Style
-- Read our official [JavaScript and React style guide](STYLE.md). Please refer to our Style Guide before asking for a review.
-- We have nothing against Prettier or any other automatic style fixers, but we generally don't use them here at Expensify. Do not use Prettier. The style changes these tools enforce don't always align with the ones we recommend and require in our eslint configs and can result in uncessary changes for our reviewers. Ignoring this advice will ultimately make your changes take longer to review as we will ask you to undo any style changes that are not relating to the important changes you are making.
+We hire and pay external contributors via Upwork.com. If you'd like to be paid for contributing, please create an Upwork account and apply for a job in the [Upwork issue list](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2). Payment for your contributions will be made no less than 7 days after the pull request is merged to allow for regression testing. We hire one contributor for each Upwork job. New contributors are limited to working on one job at a time.
```

Comment:
Is it worth mentioning explicitly that more experienced contributors are allowed to handle simultaneous jobs?

Reply:
Yep, I'll clarify that!